### DISCUSSION TITLE
Allow downloads of tracks that are requested & unavailable using Lidarr

### DISCUSSION THREAD

**Discussion by blokin** [Alee14/DLAP#11] [Ideas]
With YouTube continually removing music bots, I've been looking for an alternative.  This project has me interested but ideally instead of only having a list of specific tracks to play, I'd like users to be able to request whatever song they like and if it is not available have it added to monitoring in [Lidarr](https://github.com/Lidarr/Lidarr).

**Comment by Alee14** [comment]
Will take note for this in a future update, though not sure how to implement it myself :P

**Reply by blokin** [reply]
That'd be awesome!  I know there's an API so I don't think it would be too much to add the functionality, but I'm a noob so /shrug
